### PR TITLE
Apps Landing: Refactored scss to fix ie11 spacing issue

### PR DESCRIPTION
**Pull request by jeffgolenski**
Fixes: https://github.com/Automattic/jetpack/issues/6093

Spacing issues within IE11. I refactored some of the SCSS and also cleaned up some improper naming conventions. If any SCSS looks funky—IE made me do it.

Before: 
screen-shot-2017-01-12-at-9-18-01-pm

After:
screen shot 2017-01-16 at 4 25 43 pm

### CODE REVIEW

File: _inc/client/apps/style.scss [hunk 1]
```diff
@@ -160,50 +160,55 @@
 	}
 }
 
-.jp-landing-apps__feature-col {
+.jp-landing-apps__feature {
 
 	@include breakpoint( ">660px" ) {
-		flex-basis: 50%;
 
-		@include breakpoint( ">660px" ) {
```
Comment (MichaelArestad) [inline _inc/client/apps/style.scss]: Was there a breakpoint inside a breakpoint??

**Comment by MichaelArestad** [comment]
Sass looks good.

File: _inc/client/apps/style.scss (hunk 1 above)
Comment (jeffgolenski) [inline _inc/client/apps/style.scss]: Nah. I just think looking at the diff within the browser makes it look that way.

**Comment by jeffgolenski** [comment]
@ebinnion Can you give this a quick glance and let me know if it solves the problem on your end? IE is good to go in my testing environments.

File: _inc/client/apps/style.scss (hunk 1 above)
Comment (MichaelArestad) [inline _inc/client/apps/style.scss]: There totally was in master:
 
image

Nice fix!

Comment (jeffgolenski) [inline _inc/client/apps/style.scss]: @MichaelArestad I'd be lying if I said I intentionally fixed that. But it's been cleaned up, none the less. hahaha

**Comment by ebinnion** [comment]
I just tested in IE11 in BrowserStack and it looks good to me!

**Pull request merged**